extensions/wikia/WikiFactory/SpecialWikiFactory_body.php

```diff
@@ -1154,18 +1102,14 @@ public function __construct( $part ) {
 		$this->mRequest  = $wgRequest;
 		$this->mLimit    = 25;
 		$this->mOffset   = $this->mRequest->getVal( "offset", false );
-		$this->mTitle    = Title::makeTitle( NS_SPECIAL, 'WikiFactory' );;
+		$this->mTitle    = Title::makeTitle( NS_SPECIAL, 'WikiFactory' ); ;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this empty statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1116)